### DISCUSSION TITLE
Additional local metadata source possible?

### DISCUSSION THREAD

**Discussion by npr0n** [DirtyRacer1337/Jellyfin.Plugin.PhoenixAdult#145] [Ideas]
Thank you for the awesome work you are doing!

I am currently starting to scrape all metadata available from several different pages and storing them locally in a mongoDB.
Would it be possible to add some sort of functionality to query a local (or public) source like a database? I would be open to make the database public at some point when datastructure is finished.
Even if this is not quickly done, I think that having all the metadata collected together would help in the long run.
Unfortunately I am not familiar with C, so creating pull requests is not feasible for me...

Please let me know your thoughts!

Attached is an example json
![image](https://user-images.githubusercontent.com/76623618/171034253-9e9aca60-f50b-4a39-abd4-94d855bc420e.png)

**Comment by DirtyRacer1337** [comment]
local data would be useless for most users, don't know about public, already one (https://metadataapi.net) supported by this plugin and I make separated one (https://github.com/ThePornDatabase/Jellyfin.Plugin.ThePornDB)
